Simran/Sim_C_Program/singly_linked_list.c

```diff
@@ -0,0 +1,251 @@
+#include<stdio.h>
+#include<stdlib.h>
+//#include<allocc.h>
+
+struct node
+{
+	int data;
+	struct node *next;
+
+};
+struct node *head = NULL, *newnode , *temp;
+
+void insert_list_in_beg();
+void insert_list_in_end();
+void delete_list_from_beg();
+void delete_list_from_end();
+void display_list();
+void search_list();
+
+
+
+int 
+main() { 
```

Comment:
this curlt brace should be on new line
only control statements corly brace is on same line